src/core/cubic.h

```diff
@@ -32,6 +38,11 @@ typedef struct QUIC_CONGESTION_CONTROL_CUBIC {
     //
     BOOLEAN TimeOfLastAckValid : 1;
 
+    //
+    // HyStart state.
+    //
+    HYSTART_STATE HyStartState : 2;
```

Comment:
Why isn't this next to the other hystart fields?

Reply:
Also, be very careful with bitfields and enums. Generally enums are signed values (`int`) and if you use a bitfield to hold it, the negative representation won't match the enum value...

Reply:
Was trying to save some bits by stuffing it into the extra space left in the BOOLEAN above. Moving it now.